src/backend/cgobj.c

```diff
@@ -373,6 +373,9 @@ void too_many_symbols()
 {
 #if SCPP
     err_fatal(EM_too_many_symbols, 0x7FFF);
+#elif TARGET_WINDOS // COFF
+    error(NULL, 0, 0, "more than %d sections in object file", 65279);
```

Comment:
That number was on http://llvm.org/docs/doxygen/html/Support_2COFF_8h_source.html